river_dl/train.py

```diff
@@ -56,21 +125,18 @@ def train_model(
     :param dropout: [float] value between 0 and 1 for the probability of an
     input element to be zero
     :param num_tasks: [int] number of tasks (variables_to_log to be predicted)
-    :param learning_rate_pre: [float] the pretrain learning rate
-    :param learning_rate_ft: [float] the finetune learning rate
-    :return: [tf model]  finetuned model
+    :param learning_rate: [float] the learning rate
+    :param train_type: [str] Either pretraining (pre) or finetuning (finetune)
+    :param early_stop_patience [int] Number of epochs to use for early stopping
```

Comment:
I think it'd be good to be a little more explicit here. I like the TF description of the parameter:

```suggestion
    :param early_stop_patience [int] Number of epochs with no improvement after which training will be stopped.
```